demo_serial/src/main/java/com/novoda/downloadmanager/demo/serial/MainActivity.java

```diff
@@ -40,6 +41,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
     private void setupDownloadingExample() {
         Uri uri = Uri.parse(BIG_FILE);
+        final DownloadBatch batch = new DownloadBatch("Title woo", "Description lols", BBC_COMEDY_IMAGE);
```

Comment:
yes you use the demo for testing, but when you PR thats demo code (testing is over) so it should be correct. Not another app just make another activity in the app with this.